recipes/repository.rb

```diff
@@ -37,6 +44,30 @@
 when 'rhel', 'fedora'
   include_recipe 'yum'
 
+  # Import new RPM key
+  if node['datadog']['yumrepo_gpgkey_new']
+    # gnupg is required to check the downloaded key's fingerprint
+    package 'gnupg' do
+      action :install
+    end
+
+    # Download new RPM key
```

Comment:
Makes sense, should've made the connection with: https://github.com/DataDog/chef-datadog/pull/365/files#diff-62b784dc2245888950b940c524931921R65